Earthfile

```diff
@@ -82,11 +82,10 @@ test-js:
 
     # build
     WORKDIR /sdk/topk-js
-    RUN --mount=type=cache,target=/usr/local/share/.cache/yarn/v6 \
-        yarn install
+    RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn yarn install
 
-    RUN --mount=type=cache,target=target \
```

Comment:
ideally we'd keep this one since that's the actual cache for rust complication

Reply:
I presume rust code changes in `topk-js` and running `yarn build` will produce different contents of `target`. Therefore we want to cache `target` only when we are not changing the rust code right?

Reply:
Created an issue for this: https://github.com/fafolabs/topk-sdk/issues/36